test/xbmcgui.py

```diff
@@ -37,6 +37,13 @@ def yesno(heading, line1, line2=None, line3=None, nolabel=None, yeslabel=None, a
         print('[37;100mYESNO:[35;0m [%s] [35;0m%s[0m' % (heading, line1))
         return True
 
+    @staticmethod
+    def textviewer(heading, text=None, usemono=None):
+        ''' A stub implementation for the xbmcgui Dialog class textviewer() method '''
+        heading = kodi_to_ansi(heading)
+        text = kodi_to_ansi(text)
+        print('[37;100mTEXTVIEWER:[35;0m [%s] [35;0m%s[0m' % (heading, text))
```

Comment:
Very nice. Since you know have an implementation of textviewer. You might as well call the about dialog in the test-framework, and get bonus points for coverage :-)

Reply:
You can add it to `test/test_router.py` like the existing examples.

Reply:
Will do.